pkg/skaffold/deploy/helm.go

```diff
@@ -204,6 +204,9 @@ func (h *HelmDeployer) deployRelease(ctx context.Context, out io.Writer, r lates
 	if !isInstalled {
 		args = append(args, "install", "--name", releaseName)
 		args = append(args, h.Flags.Install...)
+	} else if !h.shouldUpgradeOnChange(r) {
+		logrus.Infof("Release %s already installed...\n", releaseName)
```

Comment:
Do you think we should handle and log remote separately?

Reply:
Sorry not sure exactly what you mean. I am open to anything just needed the main feature of not upgrading remote charts.

Do you want more logging to indicate why the chart wasn't upgraded?

Reply:
Sorry, I missed this :-(  Yes, more logging: since we're changing behaviour, it would be worth including why and what can be done.